plugins/janus_audiobridge.c

```diff
@@ -8495,6 +8355,7 @@ static void *janus_audiobridge_participant_thread(void *data) {
 			g_free(mixedpkt->data);
 			g_free(mixedpkt);
 		}
+		g_usleep(2500);
```

Comment:
Was this added to avoid busy waiting?

Reply:
Yep: before, `janus_audiobridge_participant_thread` was only used for outgoing packets, and so we had a `g_async_queue_timeout_pop` that would wait 100ms at max. Since now there's two different queues to probe, we peek both and return right away if they're empty (to check if we need to need anything with the other). This way we give the thread some idle time.